.env

```diff
@@ -0,0 +1,2 @@
+VITE_BASE_URL = https://target-api-induction-v2.herokuapp.com/api/v1/users
```

Comment:
I know in this project there isn’t too much risk but maybe you could add this .env to gitignore and only add to the repo a env.example with fake variables